docs/content/en/tasks/scale/_index.md

```diff
@@ -188,7 +188,49 @@ The StatefulSet controller manages the deletion of Cassandra pods. It deletes on
 This means for example that `my-k8ssandra-dc1-default-sts-3` will be deleted before `my-k8ssandra-dc1-default-sts-2`.
 {{% /alert %}}
 
```

Comment:
This file is in the v1 part of the docs. Don't we want to update https://docs-v2.k8ssandra.io/tasks/scale?

Reply:
Yes I was planning to revamp the entire page, I just pushed the changes.